Minio/DataModel/ObjectLockRule.cs

```diff
@@ -0,0 +1,42 @@
+/*
+ * MinIO .NET Library for Amazon S3 Compatible Cloud Storage, (C) 2020 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Xml.Serialization;
+
+namespace Minio
+{
+    [Serializable]
+    [XmlRoot(ElementName = "ObjectLockRule", Namespace = "http://s3.amazonaws.com/doc/2006-03-01/")]
+    public class ObjectLockRule
+    {
+        [XmlElement("DefaultRetention")]
+        public DefaultRetention DefaultRetention { get; set; }
+        public ObjectLockRule(DefaultRetention objectLockRetentionDefault) 
+        {
+            this.DefaultRetention = objectLockRetentionDefault;
+        }
```

Comment:
why is this constructor required - if default retention is not specfied, it is null

Reply:
Taking it out.